### PR TITLE
Places tests

### PR DESCRIPTION
**Pull request by keremidarski**

### CODE REVIEW

File: Carwash/places/tests/factories.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+faker = Faker()
+
+class PlacesFactory(factory.django.DjangoModeFactory):
+    class Meta:
+        model = Places
+
+    place_name = factory.LazyAttribute(lambda _: faker.name())
+
```
Comment (strangerok1507) [inline Carwash/places/tests/factories.py:14]: check this file by "black"

**Review by strangerok1507 — CHANGES_REQUESTED** [review]

File: Carwash/places/tests/factories.py (hunk 1 above)
Comment (keremidarski) [reply Carwash/places/tests/factories.py:14]: Checked it - nothing changed.

File: Carwash/places/tests/views/test_places_views.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            'place_name': 'edited name'
+        }
+        response = self.client.put(url, data=json.dumps(data), content_type='application/json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_delete_places_when_user_is_authenticated_expect_204(self):
+        url = reverse('Places-detail', kwargs={'pk': self.place.pk})
+        response = self.client.delete(url)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
```
Comment (strangerok1507) [inline Carwash/places/tests/views/test_places_views.py:48]: no new line

File: Carwash/places/tests/factories.py (hunk 1 above)
Comment (strangerok1507) [inline Carwash/places/tests/factories.py:14]: no new line

**Review by strangerok1507 — APPROVED** [review]
really minor changes
+1

**Pull request merged**